CRM/Core/DAO/permissions.php

```diff
@@ -114,7 +114,6 @@ function _civicrm_api3_permissions($entity, $action, &$params) {
     ),
     'default' => array(
       'access CiviCRM',
-      'edit all contacts',
```

Comment:
oh jeez.  That was meant to be temporary!  Yes, that's a dumb move.  I'll correct it.